suse_migration_services/units/prepare.py

```diff
@@ -101,6 +101,28 @@ def main():
     cloud_register_metadata = os.sep.join(
         [root_path, 'var', 'lib', 'cloudregister']
     )
+    zypper_log_path = os.sep.join(
+        [root_path, 'var', 'log', 'zypper.log']
+    )
+    if os.path.exists(zypper_log_path):
+        if not os.path.exists('/var/log'):
+            log.info('Creating /var/log')
+            Path.create('/var/log')
+
+        zypper_log_message = 'Bind mounting /var/log/zypper.log'
+        log.info(zypper_log_message)
+
+        zypper_log = Command.run(
+            ['mount', '--bind', zypper_log_path, '/var/log/zypper.log'],
+            raise_on_error=False
+        )
+        if zypper_log.return_code != 0:
+            log.warning(
+                '{0} failed with stderr: {1}, stdout: {2}.'
+                .format(
+                    zypper_log_message, zypper_log.error, zypper_log.output
+                )
+            )
```

Comment:
If you consider to just pass the exception message you can rewrite this to:

```python
try:
    Command.run(
        ['mount', '--bind', zypper_log_file, zypper_host_log_file]
    )
except Exception as issue:
    log.warning(
        'Bind mounting zypper log file failed with: {0}'.format(issue)
    )
```

There is no need to log the command information because we already log all Command.run
information since I merged #160